remote_monitoring_demo/README.md

```diff
@@ -0,0 +1,34 @@
+# Kruize Remote Monitoring Demo
+
+## Goal
+The goal of this demo is to demonstrate the workflow of Kruize in Remote monitoring mode and project the recommendations generated by kruize on grafana dashboard. The [demo](./demo.py)  script creates experiments using the [Kruize Remote Monitoring REST APIs](https://github.com/kruize/autotune/tree/mvp_demo/design/MonitoringModeAPI.md) for the specified deployment in a namespace and updates the results containing the resource usage metrics for the deployment. It then fetches and displays the Kruize recommendations for each of the experiments created.
+
+## Steps
+This demo does the following:
+- Clones the required git repositories
+- Installs minikube and prometheus
+- Installs kruize and initializes the thanos and grafana containers
+- Creates an experiment by posting the input json to Kruize Monitoring REST APIs, updates the results for the experiments and fetches the Kruize recommendations
+- Metric results along with the recommendations are backfilled in thanos and launches grafana in the browser
+- User can login to grafana and search for pronosana-dashboard to view the cpu and memory recommendations provided by Kruize
+
+## Pre-req
+It expects minikube to be installed with atleast 8 CPUs and 16384MB Memory.
+**WARNING:** The script deletes any existing minikube cluster.
+
+## How do I run it?
+
+```
+# Run the Kruize monitoring demo using the below command:
+$ ./monitoring_demo.sh
```

Comment:
Now that this has been renamed, change it to `remote_monitoring_demo.sh` in all places

Reply:
Updated, thanks.